Documentation/CrossPlatform/OculusQuestMRTK.md

```diff
@@ -32,13 +32,8 @@ This data provider does **NOT** use Unity's **XR Pipeline** or **Legacy XR Pipel
 1. Download and import [Oculus Integration](https://assetstore.unity.com/packages/tools/integration/oculus-integration-82022) from the Unity Asset Store. The latest version tested to
 work is 19.1. Older versions can be found from this [archive](https://developer.oculus.com/downloads/package/unity-integration-archive/)
 
-1. Navigate to Mixed Reality Toolkit > Utilities > Oculus > Configure CSC File for Oculus. Updating the csc file filters out the obsolete warnings produced by the Oculus Integration Assets. 
-The MRTK repo contains a csc file that converts warnings to errors, this conversion halts the MRTK-quest configuration process.
-
-    ![OculusIntegrationCsc](../Images/CrossPlatform/OculusQuest/OculusIntegrationCsc.png)
-
 1. Navigate to Mixed Reality Toolkit > Utilities > Oculus > Integrate Oculus Integration Unity Modules. Doing this will update the asmdefs with definitions and references needed for the
-relevant Oculus Quest code to function.
+relevant Oculus Quest code to function. It will also Update the csc file to filter out the obsolete warnings produced by the Oculus Integration assets. The MRTK repo contains a csc file that converts warnings to errors, this conversion halts the MRTK-quest configuration process.
```

Comment:
```suggestion
relevant Oculus Quest code to function. It will also update the csc file to filter out the obsolete warnings produced by the Oculus Integration assets. The MRTK repo contains a csc file that converts warnings to errors, this conversion halts the MRTK-Quest configuration process.
```